docs/source/tutorial/production.md

```diff
@@ -3,8 +3,104 @@ title: "4. Run your graph in production"
 description: Start here for the Apollo fullstack tutorial
 ---
 
+<h2 id="deploy">Deploy your graph API</h2>
+
+There are several services you can deploy your graph API to such as Heroku, Amazon Lambda, Netlify and Zeit's Now.  We'll deploy our graph API to Zeit's **Now**. Now is a service by Zeit that allows the deployment of an instance of Apollo Server, quickly providing a functional GraphQL endpoint.
+
+You need the following before deployment:
+
+* A [Now](https://zeit.co/now) account.
+* The [Now CLI](https://zeit.co/download#now-cli).
+
+Now looks for a `start` script in the `package.json` file to start the app. Once the script is present, Now will be able to start the app.
+
+Go ahead and run the `now` command from the root directory of the app.
+
+```bash
+$ now
+```
+
+The `now` command immediately deploys your graph API to the cloud and returns the hosted URL. The returned URL comes in this format: `<NOW_APP_NAME>.now.sh`.
```

Comment:
Isn't there a configuration setting to specify the app name? If so, let's use that.

Reply:
There isn't. It's a random generated unique name because we are using a free plan and the readers too would be using that.